packages/message-backend/src/email/create/templates/subscriptionContract.hbs

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+</head>
+<body>
+    Tilaussopimus liitteenä
```

Comment:
Should this text to be translated to different languages like in the other emails?

Reply:
This text itself is only a placeholder. Suppose the actual contents/translations are defined later